test/game_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule GameTest do
+  use ExUnit.Case
+  doctest Game
+
+  import Game
+  import Board
+
+  @empty_board empty()
+  @cats_game_board %{ 1 => "X", 2 => "O", 3 => "X", 4 => "O", 5 => "X", 6 => "O", 7 => "O", 8 => "X", 9 => "O"}
+
+  describe "Game.current_player" do
+    board = @empty_board
+    board = update(board, 5, "X") |> update(8, "O") |> update(2, "X")
+    
+    test "it returns 'O' when the turn count is odd" do
+      assert "O" == current_player(unquote(Macro.escape(board)))
+    end
+
+    board = update(board, 6, "O")
```

Comment:
We should avoid shared test data here, and instead have each test setup the date in the required state. I'm not sure how the test runner is working here, but I believe this could result in flakey tests if we used some random ordering.